testsuite/features/support/http_client.rb

```diff
@@ -13,7 +13,7 @@ def initialize(host)
   def prepare_call(name, params)
     short_name = name.split('.')[-1]
     call_type =
-      if short_name.start_with?('list', 'get', 'is', 'find') ||
+      if short_name.start_with?('list', 'get', 'is', 'find', 'system') ||
```

Comment:
shouldn't you then remove the test for `system.search` just below?

or is your new test duplicate and therefore useless?